manifests/puppetmaster.pp

```diff
@@ -37,16 +37,19 @@
 
   if $reports {   # foreman reporter
 
-    exec { 'Create Puppet Reports dir':
-      command => "/bin/mkdir -p ${puppet_basedir}/reports",
-      creates => "${puppet_basedir}/reports",
-    }
-    file {"${puppet_basedir}/reports/foreman.rb":
-      mode    => '0644',
-      owner   => 'root',
-      group   => '0',
-      source  => "puppet:///modules/${module_name}/foreman-report_${report_api}.rb",
-      require => Exec['Create Puppet Reports dir'],
+    if ( $::foreman::params::aio_package == false ) {
```

Comment:
This can be written as ```unless $::foreman::params::aio_package``` but I think it still breaks our use case. Like [I mentioned](https://github.com/theforeman/puppet-foreman/issues/390#issuecomment-289026125), we deploy it normally through puppet-puppet and set the correct ruby path there. In our installer use case we don't actually deploy the puppet module onto the puppet server (we've always stayed away from managing the users puppet environment) so no reporting ends up being installed.